### PR TITLE
MainTitle

### PR DESCRIPTION
**Pull request by JaqKent**
## Summary

Main Title 

## Screenshots

image

## Trello Card

https://trello.com/c/KI6QlhJS

### CODE REVIEW

File: src/components/MainTitle/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+/* import { useTranslation } from 'react-i18next'; */
```
Comment (nathan-hyan) [inline src/components/MainTitle/index.tsx]: Remove comments

File: src/components/MainTitle/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import style from './styles.module.scss';
+
+interface Props {
+  text: string;
+  className: string;
+}
+
+function MainTitle({ text, className }: Props) {
+  /* const { t } = useTranslation('Whoweare'); */
```
Comment (nathan-hyan) [inline src/components/MainTitle/index.tsx]: same here

File: src/components/MainTitle/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+@import "scss/variables/_colors";
+@import "scss/variables/_mixins";
+
+.container {
+  @include flex();
+}
+.title {
+  color: darken($gray, 25);
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
```
Comment (nathan-hyan) [inline src/components/MainTitle/styles.module.scss]: `font-family: 'Roboto'`

File: src/screens/Home/index.tsx [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
 import styles from './styles.module.scss';
 
 function Home() {
-  const { t } = useTranslation('Home');
+  /* const { t } = useTranslation('Home'); */
 
-  return <h1 className={styles.foo}>{t('foo')}</h1>;
+  return <MainTitle className={styles.title} text="¿Quienes Somos?" />;
 }
 export default Home;
```
Comment (nathan-hyan) [inline src/screens/Home/index.tsx]: Replace this

**Review by nathan-hyan — CHANGES_REQUESTED** [review]

File: src/components/MainTitle/i18n.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import i18next from 'i18next';
+
+i18next.addResources('es', 'WhoWeare', {
+  title: '¿Quienes somos?'
+});
+
+i18next.addResources('en', 'WhoWeare', {
+  title: 'Who We are?'
+});
```
Comment (nathan-hyan) [inline src/components/MainTitle/i18n.ts]: Delete this file

File: src/components/MainTitle/index.tsx (hunk 1 above)
Comment (JaqKent) [reply src/components/MainTitle/index.tsx]: done

File: src/components/MainTitle/index.tsx (hunk 2 above)
Comment (JaqKent) [reply src/components/MainTitle/index.tsx]: done

File: src/components/MainTitle/styles.module.scss (hunk 3 above)
Comment (JaqKent) [reply src/components/MainTitle/styles.module.scss]: done

File: src/screens/Home/index.tsx (hunk 4 above)
Comment (JaqKent) [reply src/screens/Home/index.tsx]: done

**Review by nathan-hyan — APPROVED** [review]
🥇

**Pull request merged**